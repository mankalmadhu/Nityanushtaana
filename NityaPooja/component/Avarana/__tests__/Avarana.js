import 'react-native';
import React from 'react';
import ಆವರಣ from '../Avarana';

import Renderer from 'react-test-renderer';

test("ಆವರಣ" , () => {

expect(Renderer.create(
    <ಆವರಣ>
      Avarana Texts
    </ಆವರಣ>
).toJSON()).toMatchSnapshot();

});