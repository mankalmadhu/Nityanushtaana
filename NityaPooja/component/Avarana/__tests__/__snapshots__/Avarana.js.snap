// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಆವರಣ 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  Avarana Texts
</View>
`;
